sw/device/lib/testing/test_framework/ottf_main.h

```diff
@@ -7,8 +7,8 @@
 
 #include <stdbool.h>
 
-// This private header is included here so that OTTF users can include a single
-// header in their test application (the `ottf_main.h` header).
+#include "external/freertos/include/FreeRTOS.h"
+#include "external/freertos/include/task.h"
```

Comment:
nit: including these headers makes the symbols in them available to clients, thus `kOttfFreeRtosMinStackSize` is more like an alias. Do you need the new functions to be inline, e.g. are the functions they call inline as well? If not, maybe they can be non-inline functions.

Reply:
The original intent of the inline nature of these wrapper functions was to reduce the overhead of these additional function calls in tests. As we discussed offline, moving these to non-inline functions is probably better for isolation of FreeRTOS code. Fixed in #15249.